src/python/pants/auth/basic_auth.py

```diff
@@ -1,44 +1,7 @@
 # Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from collections import namedtuple
-from dataclasses import dataclass
-from typing import Dict, Optional
-
-import requests
-import www_authenticate
```

Comment:
This can now be removed from `requirements.txt`.

Reply:
removed